src/components/graph/graph.renderer.jsx

```diff
@@ -141,6 +141,53 @@ function _renderDefs() {
  */
 const _memoizedRenderDefs = _renderDefs();
 
+/**
+ * Generates the pattern defs for drawing the grid lines.
+ * This pattern is rendered inside of a rect in the graph.
+ *
+ * @param {Object} config - an object containing rd3g consumer defined configurations {@link #config config} for the graph.
+ * @param {Object} transform - object that represents current graph transform
+ * @returns {Object} Pattern defs for gridlines. Returns empty object if gridlines are disabled.
+ */
+const _gridLineDefs = (config, transform) => {
+  const { renderGridLines, spacingX, spacingY, color, smallStrokeSize, largeStrokeSize } = config.grid;
+
+  // grid lines turned off; return nothing
+  if (!renderGridLines) {
+    return {};
+  }
+
+  const width = spacingX * transform.k;
+  const height = spacingY * transform.k;
+
+  return (
+    <>
+      {/* This is the "inner" grid lines that are thinner than the outer ones */}
+      <pattern
+        id={CONST.GRID_LINES_SMALL_GRID_PATTERN_ID}
+        width={width / 2}
+        height={height / 2}
```

Comment:
Currently, this will only ever draw two "inner" grid lines at the smaller stroke size.

I had thought about scaling the number up/down along with the zooming scale factor but that seemed a little complicated (and difficult to balance with the configurable stroke widths)